x-pack/plugins/canvas/canvas_plugin_src/uis/transforms/formatnumber.ts

```diff
@@ -4,13 +4,15 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { TransformFactory } from '../../../types/transforms';
-import { Arguments } from '../../functions/common/formatnumber';
+import { ArgumentTypeDefinitionFactory } from '../../../types';
+import { formatnumber as FormatNumberFunctionDefinitionFactory } from '../../functions/common/formatnumber';
```

Comment:
nit: this is a `const` aliased like a `type`.  The first letter should be lowercase to denote a value, rather than a TS type.